backend/app/services/knowledge/orchestrator.py

```
@@ -1396,6 +1396,239 @@ def update_document_content(
+                context_service.overwrite_attachment(
+                    db=db,
+                    context_id=document.attachment_id,
+                    user_id=user.id,
+                    filename=filename,
+                    binary_data=binary_data,
+                )
+                logger.info(
+                    f"[Orchestrator] Overwrote attachment {document.attachment_id} "
+                    f"for document {document.id}"
+                )
+            except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)